crengine/src/lvdocview.cpp

```diff
@@ -3798,13 +3799,25 @@ bool LVDocView::LoadDocument(const lChar16 * fname, bool metadataOnly) {
 				(int) stream->GetSize()));
 		m_doc_props->setString(DOC_PROP_FILE_NAME, arcItemPathName);
         m_doc_props->setHex(DOC_PROP_FILE_CRC32, stream->getcrc32());
-        const CRFileHistRecord* record = m_hist.getRecord( filename16, stream->GetSize() );
```

Comment:
Possible NULL pointer dereference. getRecord() can return NULL.

Reply:
convertBookmarks flag will be false when record is NULL.

Reply:
Ok, you right.